app/models/lit/incomming_localization.rb

```diff
@@ -36,39 +36,27 @@ def accept
     def duplicated?(val)
       set_localization
       return false if localization_has_changed?
-      translated_value =
-        localization.read_attribute_before_type_cast('translated_value')
-      if localization.is_changed? && !translated_value.nil?
-        translated_value == val
-      else
-        localization.read_attribute_before_type_cast('default_value') == val
-      end
+
+      localization.translation == val
```

Comment:
Ok I get it that we should deserialize values before comparison. However, looking at old diff it looks like there was some extra logic there. I do not mean the nil check (it is not necessary now I guess), I mean the different left-side comparison attribute depending on `is_changed?` result. Why we no longer check against default value?